test/integration-test/reference_iir/copy_stencil.iir

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
Can you put a README somewhere here how to update the reference (or a very simple bash script)